File: Python/creditCard.py
from payment import Payment
from datetime import date

class creditCard(Payment):
    entity = str
    expDate = date.today()
    cvv = int

    def __init__(self, id, entity, expDate, cvv):
        super().__init__(id)
        self.entity = entity
        self.expDate = expDate
        self.cvv = cvv

File: PHP/creditCard.php
<?php
require_once('payment.php');

class CreditCard extends Payment{
    public $entity;
    public $expDate;
    public $cvv;

    public function __construct($id, $entity, $expDate, $cvv) {
        parent::__construct($id);
        $this->entity = $entity;
        $this->expDate = $expDate;
        $this->cvv = $cvv;

    }


}
?>